resources/views/frontend/needs/add.blade.php

```diff
@@ -34,6 +34,21 @@
                             <textarea class="form-control" rows="4" id="address" name="address" required>{{ old('address') }}</textarea>
                         </div>
                     </div>
+                     <div class="form-group">
+                        <label for="latlon" class="col-lg-2 control-label">ඔබ සිටින ස්ථනය සිතියමෙන්</label>
+                        <div class="col-lg-10">
+                            <input type="hidden" class="form-control" id="latlon" name="latlon"  value="{{ old('latlon') }}"/>
+                              <button type="button" onclick="getLocation()" class="btn btn-primary">සිතියමට ඇතුල් කරන්න</button>
+                            <p id="demo">&nbsp;</p>
+                                <div id="mapholder" style="width:55%;height:300px">
+                                </div>
```

Comment:
Can we use the Maps API to get the general area from the address input above?

https://developers.google.com/maps/documentation/geocoding/intro